temporalcli/commands.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"go.temporal.io/sdk/temporal"
```

Comment:
Usually go fmt will get these non-stdlib imports separated from the stdlib ones (here and elsewhere in PR)

Reply:
It doesn't for me, for whatever reason.

Reply:
Hrmm, maybe it was `goimports` or similar that I just assumed was in `go fmt` by now